MVCapplication/src/test/java/com/softserve/if072/mvcapp/service/PdfCreatorServiceTest.java

```diff
@@ -0,0 +1,141 @@
+package com.softserve.if072.mvcapp.service;
+
+import com.softserve.if072.common.model.History;
+import com.softserve.if072.common.model.User;
+import com.softserve.if072.common.model.dto.HistorySearchDTO;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
+
+/**
+ * The PdfCreatorServiceTest class is used to test
+ * PdfCreatorService class methods.
+ *
+ * @author Vitaliy Malisevych
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class PdfCreatorServiceTest {
+
+    @Mock
+    private UserService userService;
+    @Mock
+    private RestTemplate restTemplate;
+    @Mock
+    private FileInputStream inputStream;
+
+    @InjectMocks
+    private PdfCreatorService pdfCreatorService;
+
+    @Rule
+    public TemporaryFolder folder = new TemporaryFolder();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "TemporaryFolder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2924)